scripts/aura.py

```diff
@@ -0,0 +1,82 @@
+"""
+A script to calculate % portions of user balance per given snapshot
+for FBVECVX, Rari and BVECVX LP Pool
+"""
+
+import math
+from decimal import Decimal
+from typing import Dict
+
+import simplejson as json
+from rich.console import Console
+
+from config.constants import addresses
+from helpers.enums import Network
+from rewards.boost.boost_utils import get_bvecvx_lp_ppfs
+from rewards.boost.boost_utils import get_bvecvx_lp_ratio
+from rewards.classes.Snapshot import Snapshot
+from rewards.snapshot.chain_snapshot import sett_snapshot
+from rewards.snapshot.claims_snapshot import claims_snapshot
+from rewards.snapshot.token_snapshot import fuse_snapshot_of_token
+
+console = Console()
+
+
+STARTING_BLOCK = 14848251
+MAX_TOLERANCE_THRESHOLD = Decimal(0.00001)
+
+
+def snapshot_to_percentages(snapshot: Snapshot):
+    percentages = {}
+    console.print(snapshot.total_balance())
+    for address, number in snapshot:
+        percentage = snapshot.percentage_of_total(address) * 100
+        # Filter out very small amounts
+        if percentage > Decimal(0.0000001):
```

Comment:
Is this limiter too small?  What does 0.00001% equate to in $ value -> those users will never be able to claim anything is my assumption.

Reply:
Could be, but leaving that for Trit to decide

Reply:
i would argue limit of .0000001 is actually too low, i think that we are including too many people. any way to check $ value dropped to tree for example (prob not) or any other meaningful threshold?